doc/crypto/api/keys/management.rst

```diff
@@ -202,7 +202,7 @@ When creating a key, the attributes for the new key are specified in a `psa_key_
         The following conditions can result in this error:
 
         *   ``source_key`` does not have the `PSA_KEY_USAGE_COPY` usage flag.
-        *   ``source_key`` does not have the `PSA_KEY_USAGE_EXPORT` usage flag, and its storage location does not allow copying it to the target key's storage location.
+        *   ``source_key`` does not have the `PSA_KEY_USAGE_EXPORT` usage flag, and the location of ``target_key`` is outside the security perimeter of the ``source_key`` storage location.
```

Comment:
Minor: Here we use the term 'Within the security perimeter' . Further down we refer to the 'security domain', and further still 'security bounds'. This feels like it could do with some consistency.

Reply:
Thanks - I will check what term we use elsewhere in the spec, and align on that.

Reply:
Review:

* 'security context' crops in a couple of places, relating to the use of shared memory between two 'security contexts' (perhaps meaning the same as 'execution contexts' or 'address spaces')
* 'security perimeter' is used once in the SRA appendix, in an Adversarial Model description)
* 'security boundary' is used consistently in the Design goals, Sample architectures, and Security Risk Assessment to describe trust/isolation boundaries

We're going with 'boundary'